lib/src/main/kotlin/at/bitfire/ical4android/Task.kt

```diff
@@ -54,6 +54,7 @@ class Task: ICalendar() {
     val exDates = LinkedList<ExDate>()
 
     val categories = LinkedList<String>()
+    var comment: Comment? = null
```

Comment:
I'd save the comment as a plain String because we don't sync extended properties (like language) anyway. To keep it simple, I tried to use simple data types when possible, but sometimes it's not possible, then use the ical4j property.

Then we probably also don't need the "ICal4jComment" import.